ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -6,6 +6,74 @@
     to_ivy_arrays_and_back,
 )
 
+def _broadcast_pooling_helper(x, pool_dims: str = "2d", name: str = "padding"):
+    dims = {"1d": 1, "2d": 2, "3d": 3}
+
+    if isinstance(x, int):
+        return tuple([x for _ in range(dims[pool_dims])])
+
+    if len(x) == 1:
+        return tuple([x[0] for _ in range(dims[pool_dims])])
+    elif len(x) == dims[pool_dims]:
+        return tuple(x)
+    elif len(x) != dims[pool_dims]:
+        raise ValueError(
+            f"`{name}` must either be a single int, "
+            f"or a tuple of {dims[pool_dims]} ints. "
+        )
```

Comment:
I don't think we need this helper, since our backend specific functions for `max_pool1d` already handle broadcasting. Thoughts @AnnaTz ?

Reply:
I don't think it's needed either. 
@arethel could you try removing all the extra handling and testing the function with a single `ivy.max_pool1d` call? You should add the broadcasting only If you then see errors related to the format of the arguments. 
I see you have followed the approach of the existing paddle and torch frontend pooling functions but I think those need some refactoring and we shouldn't reproduce unnecessary checks.